### PR TITLE
refactor(cmd_available/Get-Command): Add Test-CommandAvailable function

### PR DESCRIPTION
**Pull request by niheaven**
A 5 years old function `cmd_available` is aiming to check existence of `$cmd` (used to check 7z, innounp), and can replace `Get-Command` with a `[Boolean]` output.

### CODE REVIEW

File: lib/core.ps1 [hunk 1]
```diff
@@ -152,6 +152,11 @@ function file_path($app, $file) {
     return $null
 }
 
+function cmd_available($cmd) {
+    try { Get-Command $cmd -ea stop | out-null } catch { return $false }
+    $true
```
Comment (Ash258) [inline lib/core.ps1]: Why not just one liner with bool cast??

```suggestion
    return [bool] (Get-Command $cmd -ErrorAction SilentlyContinue)
```

Or at least use return
```suggestion
    return $true
```
image

Comment (niheaven) [reply lib/core.ps1]: Thanks, just move this function from `install.ps1`.

**Comment by niheaven** [comment]
Change function body to `return [Boolean](Get-Command $cmd -ErrorAction Ignore)`

**Comment by r15ch13** [comment]
Since it gets refactored anyway, let's rename it to `Test-CommandAvailable` to make a small move towards common PowerShell naming conventions :smile:

**Comment by niheaven** [comment]
Rename to `Test-CommandAvailable`

I'll rename functions in #3204 later.

File: test/Scoop-Core.Tests.ps1 [hunk 2]
```diff
@@ -135,12 +135,6 @@ Describe "get_app_name_from_ps1_shim" -Tag 'Scoop' {
         get_app_name_from_ps1_shim "$shim_path" | should -be "mockapp"
     }
 
-    It "returns app name if file exists and is a shim to an app cerca August 2018" -skip:$isUnix {
```
Comment (r15ch13) [inline test/Scoop-Core.Tests.ps1]: Why was this test removed? 🤔

Comment (niheaven) [reply test/Scoop-Core.Tests.ps1]: `shim` is tested in the above test, and this should be removed (maybe after 2018.08?)

Comment (niheaven) [reply test/Scoop-Core.Tests.ps1]: If it is not proper to remove it here, I could make a new PR to remove it. I remove it in passing while modifying the following two tests.

**Comment by r15ch13** [comment]
Please resolve the merge conflict.

**Pull request merged**